hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/event/Event.java

```diff
@@ -46,6 +56,10 @@
     @JsonInclude
     protected long ctime;
 
+    // category of Event, suitable for display, recommended to be, but not limited to, an EventCategory.name.
+    @JsonInclude
+    private String category;
```

Comment:
+1
Having an string can give it flexibility to use own "types".
An enum might be difficult to maintain.